DigitalLearningSolutions.Web/Styles/myAccount/myAccount.scss

```diff
@@ -18,11 +18,27 @@
   margin-bottom: 0px;
 }
 
-.profile-picture__label{
-    vertical-align: top;
+.account-details-list {
+  @include govuk-media-query($until: tablet) {
+    margin-bottom: 0px;
+  }
+}
+
+.account-detailslist_row:last-child {

```

Comment:
Does this do anything?

Reply:
It's to make sure these match, it has a bottom margin by default
![image](https://user-images.githubusercontent.com/59561751/116393015-0b2bca00-a819-11eb-9e64-708292ac891e.png)

Reply:
Oh, found the things it selects - must have missed them on the first pass.

Just on naming the classes in the MyAccount view - there are a few like account-detailslist_row and account-summarylist_row where the dashes don't line up with the class on the containing element (e.g. account-details-list). It might be a tad nicer if you could match those up.
Also, are you familiar with BEM naming? https://css-tricks.com/bem-101/
We seem to be roughly following that convention, though I don't believe we've taken a conscious decision to do so. According to that you'd have an extra underscore before "row", but I'll leave it to your discretion whether you want to follow that. :)

Reply:
I just followed the naming convention started by the account-summary (I didn't do that part), should be easy enough to just change them all so that they are a bit more standard, will update them.